apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableExpectationsTest.kt

```diff
@@ -0,0 +1,49 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.api.verbs.internal.expect
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.specs.*
+import kotlin.test.Test
+
+object IterableExpectationsTest : ch.tutteli.atrium.specs.integration.AbstractIterableExpectationsTest(
+    fun0(Expect<Iterable<Int>>::toHaveElements),
+    fun0(Expect<Iterable<Int>>::notToHaveElements),
+    feature0(Expect<Iterable<Int>>::min),
+    fun1(Expect<Iterable<Int>>::min),
+    feature0(Expect<Iterable<Int>>::max),
+    fun1(Expect<Iterable<Int>>::max),
+    fun0(Expect<Iterable<Int>>::toHaveElementsAndNoDuplicates),
+    property(Expect<Iterable<Int>>::last),
+    fun1(Expect<Iterable<Int>>::last),
+) {
+
+    @Test
+    fun ambiguityTest() {
+        val a1 = expect(listOf(1.0, 2.0))
+        val a1b = expect(setOf(1.0, 2.0, null))
+        val star = expect(listOf(1, 2, 3) as Collection<*>)
+
+        a1.toHaveElements()
```

Comment:
its quite some time ago where we had kotlin type inference issues (bugs) where kotlin (I think kotlin 1.3.x) inferred a wrong type and we were not be able to assign this expression to a1. Kotlin improved quite a bit since then but still the ambiguityTest is meant to detect compiler ambiguities. So I suggest we keep the assignments even when it means we need to use `var` and add a `@Suppress("AssignedValueIsNeverRead")` on the function